src/systems/implicit_system.C

```diff
@@ -113,6 +107,11 @@ void ImplicitSystem::init_matrices ()
   if (matrix->initialized())
     return;
 
+  // Clear any existing matrices
```

Comment:
> how do I run libmesh test locally?

`make check` in the top level of the build directory will run all the libmesh unit tests and examples. You can also run `make check` within a single example directory to test it alone.